cloudfoundry/config-server/create.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+source ../common.sh
+
+cf create-service -c '{"git": { "uri": "https://github.com/ilayaperumalg/config-repo"}}' $CONFIG_SERVER_SERVICE_NAME $CONFIG_SERVER_PLAN_NAME cloud-config-server
```

Comment:
That repo reference would have to change. Why not put the config needed in this very repo (in a branch even)?